Updated text + filled in blanks (#2)
greenelab/nature_news_manuscript: content/01.abstract.md

## Abstract {.page_break_before}

> Scientific news coverage shapes the public’s view of the current state of scientific findings and proposes legitimate experts.

Reviewer: ```suggestion
Scientific news coverage shapes the public’s view of the current state of scientific findings and legitimizes experts.
```

Through researching a story, journalists identify and interview a limited number of sources.
These sources may come from a journalist’s research or through recommendations by other scientists.

> In both cases, unconscious biases may influence who is identified as an expert to interview, possibly skewing the selection of interviewees.

Reviewer: ```suggestion
In either case, unconscious biases may influence who is identified as an expert to interview, possibly skewing the selection of interviewees.
```

Reviewer: this _felt_ like an either situation to me but I'm not sure - this is a soft recommendation

> To quantify possible biases in scientific news coverage, we analyzed over 22,000 news articles published by _Nature_.

Reviewer: ```suggestion
We analyzed more than 22,000 news articles published by _Nature_ to quantify possible disparities.
```

Our analysis considered three possible sources of bias: gender, name origin, and country affiliation.
To explore these sources of bias, we extracted cited authors’ names and affiliations, as well as extracted names of quoted speakers.
We then used the names to predict gender and name origin of the authors and speakers.

> In our analysis, we found a bias towards male quotation, but it is trending toward equal representation at a faster rate than academic publishing.

Reviewer: ```suggestion
In our analysis, we found a bias towards male quotation, but quotation is trending toward equal representation at a faster rate than academic publishing.
```

Interestingly, we found that the gender disparity in quotes was column-dependent, with the "career-features" column reaching gender parity.
Our name origin analysis found a significant over-representation of names with Celtic/English origin and under-representation of names with an East Asian origin.

> This finding was observed both in extracted quotes and citations.

Reviewer: ```suggestion
This finding was observed both in extracted quotes and citations in news pieces, though it was less pronounced in citations suggesting that the citation result may be driven by the selection of sources who then recommend related literature from themselves of colleagues.
```
This may be too complex for the abstract, but if we don't make this point in the discussion we probably should,

Author: Given what we discussed in our meeting earlier today, we aren't diving into the causality that may be implied by the dampened citations.
I added some clarifying sentences noting more clearly that we observe a dampened effect in citation bias.
Changes here: https://github.com/greenelab/nature_news_manuscript/pull/2/commits/007b677a21c235d26437158bb9db1fc801564a50
